### ISSUE TITLE
Error installing 1.14.0

### ISSUE THREAD

**Issue by keeneye** [bitwarden/desktop#228]
Encountered this error message on trying to install the latest update.
![Bug1](https://user-images.githubusercontent.com/1785742/54270862-40f99000-4578-11e9-97ff-8ace4f7138b8.png)

**Comment by RockGuitarist1** [comment]
I am getting this too.

**Comment by Scifire** [comment]
Same here.

**Comment by kspearrin** [comment]
See https://help.bitwarden.com/article/desktop-update-error/

TL;DR Temporarily run Bitwarden in administrator mode to let the update finish.

**Comment by Scifire** [comment]
@kspearrin what is with our users which won`t have access to admin mode?

**Comment by kspearrin** [comment]
@Scifire In that case, a reinstall would solve the problem.

**Comment by barricade823** [comment]
> 
> 
> See https://help.bitwarden.com/article/desktop-update-error/
> 
> TL;DR Temporarily run Bitwarden in administrator mode to let the update finish.

Worked for me! Thanks @kspearrin

**Comment by RockGuitarist1** [comment]
Per anyone who uses this behind a corporate firewall, Admin mode and reinstalling isn't an option. Though this isn't my case, this could be an issue for those people who fit in this category. Definitely a bug.

I will try the Admin mode when I get home from work.

**Comment by kspearrin** [comment]
@RockGuitarist1 How were you able to install the application to begin with?

**Comment by RockGuitarist1** [comment]
@kspearrin I installed it at home on my own PC, but I know that companies use password managers and could very well use Bitwarden within the confines of the company, and this would be a major issue.

**Comment by texomans** [comment]
Add another to the list. Same issue here.

**Comment by Zenexer** [comment]
Can confirm that running as admin resolves the issue.  Otherwise, I'm prompted endlessly to update, but the updates never succeed.

**Comment by texomans** [comment]
Yes, works flawlessly. Sorry, that's why I bdeleted my post right after I read the response about trying that. My apologies for not reading all posts first. Up and running great. 





Thank You,

Josh McBride, Admin

Texoma Network Solutions

P: (903) 664-5200






---- On Thu, 14 Mar 2019 01:27:26 -0500 notifications@github.com wrote ----



Can confirm that running as admin resolves the issue. Otherwise, I'm prompted endlessly to update, but the updates never succeed.

**Comment by RyanNgCT** [comment]
Having the same issue recently. I think it only happens after i toggle UAC to the recommended setting (UAC was previously off in my case)
